packages/chart/src/CdcChart.tsx

```diff
@@ -89,7 +89,7 @@ export default function CdcChart({
   const [loading, setLoading] = useState(true)
   const [colorScale, setColorScale] = useState(null)
   const [config, setConfig] = useState<ChartConfig>({} as ChartConfig)
-  const [stateData, setStateData] = useState(configObj.data || [])
+  const [stateData, setStateData] = useState(configObj?.data || [])
```

Comment:
configObj should always be defined so we should not add this ? as it passes the error down to the next component in the tree.

Reply:
I think it might be needed for standalone charts

Reply:
Fixed!